yarn.lock

```diff
@@ -4871,23 +4871,17 @@
   resolved "https://registry.yarnpkg.com/@hookform/error-message/-/error-message-0.0.5.tgz#606912b506513669980c440f383ff6c78a7aca3a"
   integrity sha512-es7eLLFA3SXNYAT8aUjvf7Gok1eMHK+9DMILtJA7ZEwYZlCCCPifhpoZmY+5SOopEtF7e+qxFOjX+MJnI3jOcg==
 
-"@hugsmidjan/qj@^4.7.0":
-  version "4.7.0"
-  resolved "https://registry.yarnpkg.com/@hugsmidjan/qj/-/qj-4.7.0.tgz#ef2793bf1f349bc01b2118dc2d20d6f22163fa84"
-  integrity sha512-/kIAN1WvMTlzmtKXgIRl57KbTpAd9UmNgPfcO0NtZNllwhfKDRVOrN9c6ZeBxnrfWsYBP6/BFldYur1aBDBO2g==
+"@hugsmidjan/qj@^4.1.1", "@hugsmidjan/qj@^4.7.0":
+  version "4.9.0"
+  resolved "https://registry.yarnpkg.com/@hugsmidjan/qj/-/qj-4.9.0.tgz#02ca1a92a3c1ba9c069b5a89bcf3a29a9e250932"
+  integrity sha512-Jsk1vOdF7HwihN1qzDK9pMADADg0x7yDCOiQMBfciK/A6q/W6O2zjBNI/jA+u70I8GgI+PG3hYgE1gSe4gizDg==
 
-"@hugsmidjan/regulations-editor@0.4.8":
-  version "0.4.8"
-  resolved "https://registry.yarnpkg.com/@hugsmidjan/regulations-editor/-/regulations-editor-0.4.8.tgz#aadf218dd8761d6c04a39dc4c4879d9d84b40588"
-  integrity sha512-+zfGUehMvgGqJAj++kcJu5V/MW9L8s5di96O6tBPMaufYX2wmNRIn9MkT+HmOS/z7LJQ47GWnSAYfdsZkhQW2w==
+"@hugsmidjan/react@^0.4.4":
+  version "0.4.5"
+  resolved "https://registry.yarnpkg.com/@hugsmidjan/react/-/react-0.4.5.tgz#3756ae5058a706cc899e4455110c19af7919372c"
+  integrity sha512-DxY6bqcXVnhYFQVAAUmrUYR7JE/199XW/hyuoz4/fsyr5CJGlhZGOuoZp9DoDKot3oU99zxLTP4uCCfZDzbR1Q==
   dependencies:
-    "@hugsmidjan/qj" "^4.7.0"
-    "@tinymce/tinymce-react" "^3.8.4"
-    cleanup-prettier "npm:prettier@2.2.0"
-    htmldiff-js "^1.0.5"
-    jsdom "^16.4.0"
-    tinymce "^5.6.2"
-    translate.js "^1.3.2"
+    "@hugsmidjan/qj" "^4.1.1"
```

Comment:
Hver er staðan á að draga þetta líka inn eða open sourca hjá ykkur með permissive license?

Reply:
> What's the status on getting rid of this, or open sourcing it on your side with a permissive license?

Both of these packages are open source and released under MIT license:
https://github.com/hugsmidjan-is/react-utils
https://github.com/hugsmidjan/qj

Reply:
Sweet! 👏